node-sdk-sample/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+package-lock.json
```

Comment:
I see that `package-lock.json` is listed here but was still committed. Should we `git rm` it from the repo?